testsuite/integration/microprofile/src/test/java/org/wildfly/test/integration/observability/micrometer/MicrometerRemovalTestCase.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.wildfly.test.integration.observability.micrometer;
+
+import java.io.File;
+import java.io.IOException;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.io.input.Tailer;
+import org.apache.commons.io.input.TailerListener;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.testcontainers.api.DockerRequired;
+import org.jboss.as.arquillian.api.ContainerResource;
+import org.jboss.as.arquillian.container.ManagementClient;
+import org.jboss.as.controller.client.helpers.Operations;
+import org.jboss.as.test.shared.ServerReload;
+import org.jboss.as.test.shared.observability.setuptasks.AbstractSetupTask;
+import org.jboss.dmr.ModelNode;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@RunWith(Arquillian.class)
+@DockerRequired
+@RunAsClient
+public class MicrometerRemovalTestCase {
+    private static final String ERROR_MESSAGE = "Failed to publish metrics to OTLP receiver";
```

Comment:
Where does this message come from? If the message is internationalized, this could cause the test to fail.

Reply:
That come from Micrometer. The full error message is "Failed to publish metrics to OTLP receiver (context: url=http://localhost:4318/v1/metrics/v1/metrics, resource-attributes={service.name=wildfly}): java.net.ConnectException: Connection refused". There's no code to search for, so I grabbed something that seemed small enough to be safe-ish without being likely to hit false positives.

Reply:
This is coming from directly from `io.micrometer.registry.otlp.OtlpMeterRegistry` and there is no i18n.

Reply:
@jamezp Speaking of - do we ever run our test suite with non-English locale? I am not aware that we do.

Reply:
@rhusar I don't think anything on ci.wildfly.org is setup that way. It really just depends on where it's ran TBH. I know in the past we've had some CI setup with`cs-CZ.UTF-8`.